exasol/slc/tool/commands/run_db_tests.py

```diff
@@ -149,6 +149,11 @@
 @add_options(system_options)
 @add_options(luigi_logging_options)
 @add_options(export_options)
+@click.option(
+    "--gpu-enabled/--no-gpu-enabled",
+    default=False,
+    help="""Enables GPU support in the spawned test DB container.""",
+)
```

Comment:
For simplicity I added this new parameter "gpu-enabled".
It's a shortcut for the 3 parameters needed to pass to the DB: `docker_runtime` / `additional_db_parameter` and `docker_environment_variable`.

The other option would be to add the `docker_runtime` as new option (`docker_environment_variable` and `additional_db_parameter` are already options) and let the client set the appropriate values.

Reply:
in general, I like the idea, but the settings are nvidia specific and might be different for other vendors

Reply:
I see, so maybe instead of a boolean we could have a Enum:
`accelerator=nvidia`
?

Reply:
Done